### PR TITLE
AGW: pipelined: qos: skip filter for APN ambr queue

### PR DESCRIPTION
**Pull request by pshelar**
QoS does not use apn root qos filter, avoid creating it.
This patch also increases FD limits on pipelineD service
this helps in scale test.

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/python/magma/pipelined/qos/qos_meter_impl.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def add_qos(self, _, qos_info: QosInfo, parent=None) -> int:
         LOG.debug("Adding meter_id %d", meter_id)
         return meter_id
 
-    def remove_qos(self, meter_id: int, d, recovery_mode=False):
+    def remove_qos(self, meter_id: int, d, recovery_mode=False, _=False):
```
Comment (koolzz) [inline lte/gateway/python/magma/pipelined/qos/qos_meter_impl.py:75]: _=False -> _

File: lte/gateway/deploy/roles/magma/files/systemd/magma_pipelined.service [hunk 2]
```diff
@@ -21,6 +21,9 @@ Wants=openvswitch-switch.service
 [Service]
 Type=simple
 PermissionsStartOnly=true
+LimitNOFILE=49000
```
Comment (koolzz) [inline lte/gateway/deploy/roles/magma/files/systemd/magma_pipelined.service]: why specifically 49000? maybe just 50000

**Review by koolzz — APPROVED** [review]
lgtm

File: lte/gateway/python/magma/pipelined/qos/qos_meter_impl.py (hunk 1 above)
Comment (pshelar) [reply lte/gateway/python/magma/pipelined/qos/qos_meter_impl.py:75]: it needs default value, since recovery_mode param has it.

**Review by ardzoht — APPROVED** [review]

**Pull request merged**